socorro/unittest/schemas/test_schemas.py

```diff
@@ -25,21 +26,467 @@ def test_validate_schemas(reporoot):
+    "upload_file_minidump_browser.json_dump.modules.[].end_addr",
+    "upload_file_minidump_browser.json_dump.modules.[].filename",
+    "upload_file_minidump_browser.json_dump.modules.[].loaded_symbols",
+    "upload_file_minidump_browser.json_dump.modules.[].missing_symbols",
+    "upload_file_minidump_browser.json_dump.modules.[].symbol_url",
+    "upload_file_minidump_browser.json_dump.modules.[].version",
+    "upload_file_minidump_browser.json_dump.modules_contains_cert_info",
+    "upload_file_minidump_browser.json_dump.pid",
+    "upload_file_minidump_browser.json_dump.status",
+    "upload_file_minidump_browser.json_dump.system_info",
+    "upload_file_minidump_browser.json_dump.system_info.cpu_arch",
+    "upload_file_minidump_browser.json_dump.system_info.cpu_count",
+    "upload_file_minidump_browser.json_dump.system_info.cpu_info",
+    "upload_file_minidump_browser.json_dump.system_info.cpu_microcode_version",
+    "upload_file_minidump_browser.json_dump.system_info.os",
+    "upload_file_minidump_browser.json_dump.system_info.os_ver",
+    "upload_file_minidump_browser.json_dump.thread_count",
+    "upload_file_minidump_browser.json_dump.threads",
+    "upload_file_minidump_browser.json_dump.threads.[]",
+    "upload_file_minidump_browser.json_dump.threads.[].frame_count",
+    "upload_file_minidump_browser.json_dump.threads.[].frames",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[]",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].file",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].frame",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].function",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].function_offset",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].line",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].missing_symbols",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].module",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].module_offset",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].offset",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].registers",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].registers.(re:^.+$)",
+    "upload_file_minidump_browser.json_dump.threads.[].frames.[].trust",
+    "upload_file_minidump_browser.json_dump.threads.[].last_error_value",
+    "upload_file_minidump_browser.json_dump.threads.[].thread_name",
+    "upload_file_minidump_browser.mdsw_return_code",
+    "upload_file_minidump_browser.mdsw_status_string",
+    "upload_file_minidump_browser.stackwalk_version",
+    "upload_file_minidump_browser.success",
+    "uptime",
+    "uptime_ts",
+    "useragent_locale",
+    "utility_process_sandboxing_kind",
+    "uuid",
+    "vendor",
+    "version",
+    "windows_error_reporting",
+    "xpcom_spin_event_loop_stack",
+}
+
+
 def test_processed_crash_schema():
```

Comment:
I should split the processed crash schema tests into a separate module. It's weird to have one module testing all the schema things.